test/pummel/test-keep-alive.js

```diff
@@ -104,7 +104,19 @@ server.listen(common.PORT, () => {
 });
 
 process.on('exit', function() {
-  assert.strictEqual(true, normalReqSec > 50);
-  assert.strictEqual(true, keepAliveReqSec > 50);
-  assert.strictEqual(true, normalReqSec < keepAliveReqSec);
+  assert.strictEqual(
+    normalReqSec > 50,
+    true,
+    `${normalReqSec} should be greater than 50`
+  );
```

Comment:
thanks for the effort to put in this additional message. However, it can be improved a little more, to be of great value:
For example, consider `normalReqSec` to be 30. Now with the change, the assertion would be:

`AssertionError [ERR_ASSERTION]: 30 should be greater than 50`

Instead, it would be nice to see like:
`AssertionError [ERR_ASSERTION]: normalReqSec should be greater than 50, but got 30`

to achieve that, please modify the third parameter as:
`normalReqSec should be greater than 50, but got ${normalReqSec}`

Reply:
Thank you @gireeshpunathil for the suggestion and @thefourtheye for making the change.

Reply:
I guess the problem with this is that if I get the right value I'll have a message that reads weird.

"AssertionError [ERR_ASSERTION]: 100 should be greater than 50, but got 100."

Reply:
@n370 I think the idea would be to have the first `normalReqSec` be the literal string `normalReqSec`, not a placeholder? :)

(I’d consider this comment optional, btw – the relevant information is present, and together with the stack trace diagnosing a failing assertion here should be very easy)

Reply:
@addaleax @gireeshpunathil I think I get it now. I can make the improvement if you really want.

```
  assert.strictEqual(
    normalReqSec > 50,
    true,
    `normalReqSec should be greater than 50, but got ${normalReqSec}`
  );
  assert.strictEqual(
    keepAliveReqSec > 50,
    true,
    `keepAliveReqSec should be greater than 50, but got ${keepAliveReqSec}`
  );
  assert.strictEqual(
    normalReqSec < keepAliveReqSec,
    true,
    `${normalReqSec} should be less than ${keepAliveReqSec}`
  );
```

👍 if you want me to update
👎 if you think it is unnecessary overhead

Reply:
the 3rd statement would be:
`normalReqSec should be less than ${keepAliveReqSec}, but got ${normalReqSec }`

Reply:
What about the following?

normalReqSec should be less than keepAliveReqSec, but ${normalReqSec} is greater than ${keepAliveReqSec}

I'm just not sure about how to make it respecting the 80char limit.

Reply:
Adding a line break inside a template string actually adds a line break to the interpolated result so I'm not sure how to format the code if we're to move forward with that change as well.

Reply:
I guess you could break the template string into 2 as shown below:

$ cat a.js
```js
var x = 10;
var y = 20;
console.log(`hello ${10} ` +
            `and ${y}`);
```

$ node a.js
hello 10 and 20
$

To be clear, I am not persisting on my way of doing things, you may chose to leave it as it is; or address this; with no strong opinion on either way: Leaving it out for this PR has the advantage of resolving the PR quickly, addressing this has an advantage of going through the practical nuances of a PR, so!

Reply:
@n370 I’d suggest just going with whatever *you* think is best… :)

Reply:
Thanks @gireeshpunathil, I'm using your suggestion to break into multiple lines. I just had to use a single quote string for the first operand because of:

```
Running JS linter...
node/test/pummel/test-keep-alive.js
  120:5  error  Strings must use singlequote  quotes

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.

make[1]: *** [lint-js] Error 1
```

@addaleax also thank you for the kind words.